litgpt/config.py

```diff
@@ -1530,6 +1530,42 @@ def norm_class(self) -> Type:
         intermediate_size=14336,
     )
 )
+configs.append(
+    # https://huggingface.co/mistralai/Mistral-7B-v0.3/blob/main/config.json
```

Comment:
From what repo this model is: unsloth or Mistral?
The link is to Mistral, but the org field in the config is unsloth.

Reply:
Just solved this in the new commit